app/views/multifactor_auths/new.html.erb

```diff
@@ -7,9 +7,9 @@
   <div>
     <p><%= t('.scan_prompt') %></p>
     <p>
-      <span id="mfa-account"><%= t('.account', account: "#{issuer}:#{current_user.email}") %></span><br>
-      <span id="mfa-key"><%= t('.key', key: @seed.chars.each_slice(4).map(&:join).join(' ')) %></span><br>
-      <span id="mfa-type"><%= t('.time_based') %></span>
```

Comment:
Do you know what these ids are used for? If they aren't used for anything, I think we can scrap them.

Reply:
`[mfa/otp]-key` is used in a handful of tests within multifactor_auths_controller_test.rb and settings_test.rb, I suppose there may be another way to do it without but I left it as is for now but removed the `mfa-account` and `mfa-type` ids!